Documentation/docfx.json

```diff
@@ -23,7 +23,7 @@
         "src": ".."
       },
       {
-        "files": [ "tutorial/**/*.{md,yml}", "spec/**/*.{md,yml}", "*.{md,yml}", "guideline/**/*.{md,yml}", "templates-and-plugins/**/*.{md,yml}" ]
+        "files": [ "tutorial/**/*.{md,yml}", "spec/**/*.{md,yml}", "*.{md,yml}", "guideline/**/*.{md,yml}", "templates-and-plugins/**/*.{md,yml}", "incrementalbuild/**/*.{md,yml}" ]
```

Comment:
how about tutorial since user might need to reference it to customize their processor?